Userland/Libraries/LibC/arpa/inet.cpp

```diff
@@ -27,8 +27,8 @@ char const* inet_ntop(int af, void const* src, char* dst, socklen_t len)
             errno = ENOSPC;
             return nullptr;
         }
-        auto str = IPv6Address(((in6_addr const*)src)->s6_addr).to_deprecated_string();
-        if (!str.copy_characters_to_buffer(dst, len)) {
+        auto str = IPv6Address(((in6_addr const*)src)->s6_addr).to_string().release_value_but_fixme_should_propagate_errors();
```

Comment:
If `to_string()` fails, that's probably `ENOMEM` right? You could just return that error directly after checking.

Reply:
Since the posix standard does not mention `ENOMEM` for [inet_ntop](https://pubs.opengroup.org/onlinepubs/9699919799.2018edition/), I was unsure if that was OK. But I changed it to set `ENOMEM` now.